src/sass/main.scss

```diff
@@ -9,6 +9,7 @@ body {
 }
 
 @import './utils/vars';
+@import './base/container';
```

Comment:
nie potrzebne import i cała plik w main jest container, jak potrzebujesz dodać dodatkowe parametry do container dajesz <div class="container jakaś tam twoja klasa albo  container container--modyfikator i jest w ten sposób zgodność z bem >lorem</div>

Reply:
Do zmiany geometrii stosuje się domieszki. Zwykle container robi się uniwersalny, czyli dla miejsc gdzie najwięcej razy się powtarza dana zależność. A nadpisuje się tylko jako wyjątek, np.
 <header>
      <div class="page-heder">
        <div class="container page-heder__container">

Reply:
Pozatym container robiłem według szkolenia z platformy. https://www.youtube.com/watch?v=GSIH3TAOeCs tam container jest jako zewnętrzny pojedynczy fragment.

Reply:
"Do zmiany geometrii stosuje się domieszki. Zwykle container robi się uniwersalny, czyli dla miejsc gdzie najwięcej razy się powtarza dana zależność. A nadpisuje się tylko jako wyjątek, np."

no właśnie i ty masz te domieszki stosujesz w kontenerze uniwersalny kontener jest w main.scss i zawiera szerokość oraz paddingi

Pozatym container robiłem według szkolenia z platformy. https://www.youtube.com/watch?v=GSIH3TAOeCs tam container jest jako zewnętrzny pojedynczy fragment.

"Programowanie nie polega na przepisywaniu bo ktoś tak zrobił, ja zimą chodzę w sandałach to ty też masz ? Jesteśmy grupa i trzeba się dostosować ta strona nie będzie działała jak każdy nazwie sobie klase container. Jest jeden uniwersalny dla wszystkich potrzebujesz do niego dodatkowe parametry nadaj inna nazwę klasy lub zastosuj modyfikator."